tests/test_isrTaskLSST.py

```diff
@@ -476,6 +476,79 @@ def test_isrFlat(self):
         ratio = result2.exposure.image.array / result.exposure.image.array
         self.assertFloatsAlmostEqual(ratio, flat_nodefects.image.array, atol=1e-4)
 
+    def test_isrBrighterFatter(self):
+        """Test processing of a flat frame."""
+        mock_config = self.get_mock_config_no_signal()
+        mock_config.doAddDark = True
+        mock_config.doAddFlat = True
+        # The doAddSky option adds the equivalent of flat-field flux.
+        mock_config.doAddSky = True
+        mock_config.doAddSiliconPhysics = True
+        mock_config.bfStrength = 1.0
+        mock_config.diffusionStrength = 0.0
```

Comment:
If we're going to override this, this should be the default.

Reply:
This is the default actually :)